pkg/operator/ceph/cluster/mon/config.go

```diff
@@ -258,8 +260,15 @@ func createNamedClusterInfo(context *clusterd.Context, clusterName string) (*cep
 	}
 
 	// generate the admin secret if one was not provided at the command line
-	args := []string{"--cap", "mon", "'allow *'", "--cap", "osd", "'allow *'", "--cap", "mgr", "'allow *'", "--cap", "mds", "'allow'"}
-	adminSecret, err := genSecret(context.Executor, dir, client.AdminUsername, args)
+	adminArgs := []string{"--cap", "mon", "'allow *'", "--cap", "osd", "'allow *'", "--cap", "mgr", "'allow *'", "--cap", "mds", "'allow'"}
+	adminSecret, err := genSecret(context.Executor, dir, client.AdminUsername, adminArgs)
+	if err != nil {
+		return nil, err
+	}
+
+	// generate the csi secret if one was not provided at the command line
+	csiArgs := []string{"--cap", "mon", "'profile rbd'", "--cap", "osd", "'profile rbd'"}
```

Comment:
CSI need `mgr` permission also https://github.com/ceph/ceph-csi/issues/677#issuecomment-540567370

Reply:
Will update thanks, I was just following the example given in #4074.

Reply:
cc @dillaman correct me if am wrong

Reply:
Correct -- need "allow rw" for MGR as well. I am adding a new "profile rbd" for MGR but it will take time to trickle down.

Additionally, assuming the CephFS CSI would also use this user, what caps would it need?